submissions/idiagheOsaigbovo/HtmlTask/rework.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Restration form</title>
+    </head>
+    <body>
+       <div>
+            <center>
+                <h1><strong>WELCOME TO AFRIDIRECT</strong></h1>
+                <br>  
+                <p><b> We buy and sell everything Africa</b></p>
+            </center>
+       </div>
+
+       <fieldset>
+           <legend><strong>Login</strong></legend>
+
+           <form method="POST" action="login.php">
+                <label for="email">E-mail Address* </label> <br>
+                <input type="email" name="email" id="email">
+                <br>
+                <br><label for="password">Password*</label> <br>
+                <input type="password" name="password" id="password">
+                <br>
+                <br><input type="submit" name="submit" value="LOGIN">
+                <a href="forgot password" target="_blank">Forgot Password?</a>
+           </form>
+       </fieldset>
+
+        <p> signup, <a href="AboutUs.html">Click Here</a> </p>
+
+        <fieldset>
+            
+            <legend><strong>SignUp</strong></legend>
+            <form action="results.html" method="post">
+
+                <label for="name">Name <br><input type="text" name="name
```

Comment:
Oops! It seems you forgot the closing quotes for the name attribute.

Reply:
I think the input element shouldn't be in the label element. Can you please check for the right way to use the label element.

Reply:
Also, please try to format this section properly. The elements here are quite packed and it is quite difficult to read.